EasyReflectometry/sample/layer.py

```diff
@@ -372,38 +374,39 @@ def chemical_structure(self) -> str:
         :return: Chemical structure
         """
         return self._chemical_structure
-    
+
     @chemical_structure.setter
     def chemical_structure(self, structure_string: str):
-       """
+        """
        :param structure_string: String that defines the chemical structure.
-       """ 
-       self._chemical_structure = structure_string
-       scattering_length = neutron_scattering_length(structure_string)
-       self.scattering_length_real.value = scattering_length.real
-       self.scattering_length_imag.value = scattering_length.imag
-       self.material.name = structure_string + '/' + self.material._material_b.name
+       """
+        self._chemical_structure = structure_string
+        scattering_length = neutron_scattering_length(structure_string)
+        self.scattering_length_real.value = scattering_length.real
+        self.scattering_length_imag.value = scattering_length.imag
+        self.material.name = structure_string + '/' + self.material._material_b.name
 
     @property
     def _dict_repr(self) -> dict:
         """
-        Dictionary representation of the :py:class:`LayerApm` object. 
-        
+        Dictionary representation of the :py:class:`LayerApm` object.
+
         :return: Simple dictionary
         """
         layerapm_dict = super()._dict_repr
         layerapm_dict['chemical_structure'] = self._chemical_structure
         layerapm_dict[
-            'area_per_molecule'] = f'{self.area_per_molecule.raw_value:.1f} {self.area_per_molecule.unit}'
+            'area_per_molecule'] = f'{self.area_per_molecule.raw_value:.1f} ' \
+                                   f'{self.area_per_molecule.unit}'
         return layerapm_dict
 
-    def as_dict(self, skip: list=[]) -> dict:
+    def as_dict(self, skip: list = []) -> dict:
```

Comment:
Are you sure with defaulting `list` to `[ ]`? Lists are mutable default arguments and can be tricky.
https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument